src/Meziantou.Framework.FastEnumToStringGenerator/EnumToStringSourceGenerator.cs

```diff
@@ -146,9 +148,19 @@ private static string GenerateCode(ImmutableArray<EnumToProcess> enums)
                     sb.AppendLine("{");
                 }
 
+                if (typeIsPublic)
+                {
+                    sb.AppendLine($"/// <summary>A class with memory-optimized alternative to regular ToString() on enums.</summary>");
+                }
+
                 sb.Append(typeVisibility).AppendLine(" static partial class FastEnumToStringExtensions");
                 sb.AppendLine("{");
 
+                if (enumeration.IsPublic)
+                {
+                    sb.Append("    ").AppendLine($"/// <summary>A memory-optimized alternative to regular ToString() method on {enumeration.FullCsharpName} enum.</summary>");
```

Comment:
Would be nice to use a cref tag to link the enum. There is a class named DocumentationId (I'm not in front of a computer to check the name) that you can use to generate the cref value. If it's too complicated it's ok to not fix this comment.

Reply:
I will check, thx

Reply:
Took a look and found `DocumentationCommenId` class but did not realize how to use it so I chose a naive approach. Let me know. It looks good though.

<img width="985" alt="image" src="https://github.com/meziantou/Meziantou.Framework/assets/9723402/41bb4ca8-776d-45ed-99c0-542e83916615">

Reply:
I'll check your changes when I get back from vacation next week

Reply:
Thank you

Reply:
I think you can add the `DocumentationId` property in the record `EnumToProcess`. BTW, you can replace the computed properties with `{get;} = ` to reduce some allocations.

````c#
    private sealed record EnumToProcess(ITypeSymbol EnumSymbol, List<EnumMemberToProcess> Members, bool IsPublic, string? Namespace)
    {
        public string FullCsharpName { get; } = EnumSymbol.ToString()!;
        public string? FullNamespace { get; } = Namespace ?? GetNamespace(EnumSymbol);
        public string? DocumentationId { get; } = DocumentationCommentId.CreateDeclarationId(EnumSymbol);

        // ...
    }
````

Then, you can use the new property in the `cref` attribute.

Reply:
Great, thank you, just updated the sources.